src/store/channels/api.ts

```diff
@@ -1,7 +1,13 @@
-import { get } from '../../lib/api/rest';
+import { get, post } from '../../lib/api/rest';
 
 export async function fetchUsersByChannelId(channelId: string): Promise<any> {
   const response = await get<any>(`/chatChannels/${channelId}/members`);
 
   return response.body;
 }
+
+export async function joinChannel(channelId: string): Promise<any> {
```

Comment:
can we change the any type, I know we have it already in the above method - let's try to get rid of it